pkg/printers/humanreadable.go

```diff
@@ -395,6 +408,86 @@ func PrintTable(table *metav1beta1.Table, output io.Writer, options PrintOptions
 	return nil
 }
 
+// PrintTableWithExtraCols prints a table to the provided output respecting the filtering rules for options
+// for wide columns and filtered rows and printing any extra columns using provided parsers. It filters out
+// rows that are Completed. You should call DecorateTable if you receive a table from a remote server before
+// calling PrintTableWithExtraCols.
+func PrintTableWithExtraCols(table *metav1beta1.Table, parsers []*jsonpath.JSONPath, output io.Writer, options PrintOptions) error {
```

Comment:
We could try re-using most of the existing code in the `PrintTable` func by updating its signature to receive the new `parsers` parameter and handling that slice there, instead of creating this whole new function